src/parcsr_ls/par_amg_setup.c

```diff
@@ -3067,12 +3068,12 @@ hypre_BoomerAMGSetup( void               *amg_vdata,
       }
 
       HYPRE_ANNOTATE_MGLEVEL_END(level);
-#if defined(HYPRE_USING_NVTX)
+#if defined (HYPRE_USING_GPU)
       hypre_GpuProfilingPopRange();
 #endif
       ++level;
       HYPRE_ANNOTATE_MGLEVEL_BEGIN(level);
-#if defined(HYPRE_USING_NVTX)
+#if defined(HYPRE_USING_GPU)
       hypre_sprintf(nvtx_name, "%s-%d", "AMG Level", level);
```

Comment:
@liruipeng , do you mean this `sprintf`? If yes, note it's used in the `hypre_GpuProfilingPushRange` call below